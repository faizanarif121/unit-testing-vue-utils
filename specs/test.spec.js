import { mount } from '@vue/test-utils'
import TestComponent from '@/test'

test('mount a vue component', () => {
  const wrapper = mount(TestComponent, {
    propsData: {
      value: 'Faizan Arif',
    },
  })
  expect(wrapper.html()).toMatchSnapshot()
})
