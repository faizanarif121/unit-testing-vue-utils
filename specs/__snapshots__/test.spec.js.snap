// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount a vue component 1`] = `
<div>
  <h1>Hello Faizan Arif</h1>
</div>
`;
